PeerioTechnologies/peerio-mobile
# fix: save ak to txt on signup for android without pdf reader [branch …

…ch13122]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/13122/android-improved-handling-of-ak-pdf-in-account-creation

#### Testing instructions  
There were no designs for this, but they're unnecessary. Important thing is to make sure that the flow for saving AK on an Android device without a pdf reader is the same as saving it on one that has a reader.
Flow:
1- In sign up account key backup screen, press save PDF
2- Create, save and open a txt file named `*username*-*appName*}.txt`
3- File contains labelled Username and Account Key
4- User presses back to exit the txt file and continues account creation flow

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
